jobs/update_orphaning_dashboard_etl.py

```diff
@@ -186,7 +149,7 @@ def longitudinal_shim_aggregate(date_from, date_to, destination_project, destina
             `moz-fx-data-shared-prod.telemetry.main`
         WHERE
             sample_id = 42
-            AND DATE(submission_timestamp)>='2019-09-01' )
+            AND environment.build.version IS NOT NULL )
```

Comment:
This is new, what's the rationale?

Reply:
We had a single ping with NULLed version recently which caused this break. Such pings shouldn't be included here since we're specifically looking at various versions in this analysis.